v2/public/localscripts/calculationQuestion/SimpleClient.js

```diff
@@ -248,11 +248,56 @@ define(["jquery", "underscore", "backbone", "rivets", "PrairieTemplate"], functi
             this.remove();
         },
     });
+    
+    var SubmissionView = Backbone.View.extend({
+        
+        initialize: function() {
+            this.questionDataModel = this.options.questionDataModel,
+            this.appModel = this.options.appModel,
+            this.template = this.options.template,
+            this.params = this.options.params,
+            this.thisSubmission = this.options.thisSubmission,
+            this.rivetsBindingsActive = false;
+            this.render();
+        },
+        
+        render: function() {
+            if (this.rivetsBindingsActive)
+                this.rivetsView.unbind();
+            var templateData = {
+                params: this.params.toJSON(),
+                thisSubmission: this.thisSubmission,
+            };
+            var templatedHTML = PrairieTemplate.template(this.template, templateData, this.questionDataModel, this.appModel);
+            if (this.options.templateTwice) {
+                templatedHTML = PrairieTemplate.template(templatedHTML, {}, this.questionDataModel, this.appModel);
+            }
+            this.$el.html(templatedHTML);
+            this.thisSubmissionObject = new Backbone.Model(this.thisSubmission);
+            this.rivetsView = rivets.bind(this.$el, {
+                model: this.model,
+                params: this.params,
+                submittedAnswer: this.thisSubmissionObject,
+            });
+            //this.rivetsBindingsActive = true;
+            if (window.MathJax)
+                MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
+            //this.trigger("renderFinished"); 
+            return this;
+        },
+        
+        close: function() {
+            if (this.rivetsBindingsActive)
+                this.rivetsView.unbind();
+            this.remove();
+        },
+    }); 
```

Comment:
There is trailing whitespace on this line (extra spaces after the semicolon). Github doesn't show it, but it shows up in red when I do "git diff" on the command line (at least on my machine).

There are a few other trailing-whitespace errors in this PR. The general rule is that shouldn't be trailing whitespace on any line.